File: 107598019/HW2/Software Architecture_MySQL/adapter/presenter/DeleteCoursePresenter.cs
using Software_Architecture.usecase.deleteCourse;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.adapter.presenter
{
    public class DeleteCoursePresenter : DeleteCourseOutput
    {
        public DeleteCoursePresenter() { }

        private string msg;

        public string getMessage()
        {
            return msg;
        }

        public void setMessage(string title)
        {
            this.msg = title + " is gone.";
        }
    }
}

File: 107598034/HW2/CourseSystem/src/Adapter/Presenter/DeleteCoursePresenter.java
package Adapter.Presenter;

import CourseUseCase.DeleteCourseUseCase.DeleteCourseOutput;

public class DeleteCoursePresenter implements DeleteCourseOutput {
	
	private String courseName = "";

	@Override
	public String getCourseName() {
		return courseName;
	}

	@Override
	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}

}
